UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/ISalesCatalogueService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Models.Request;
+using UKHO.ExchangeSetService.Common.Models.SalesCatalogue;
+
+namespace UKHO.ExchangeSetService.Common.Helpers
+{
+    public interface ISalesCatalogueService
+    {
+        public Task<SalesCatalogueResponse> GetProductsFromSpecificDateAsync(string sinceDateTime);
+        public Task<SalesCatalogueResponse> PostProductIdentifiersAsync(List<string> ProductIdentifiers);
+        public Task<SalesCatalogueResponse> PostProductVersionsAsync(List<ProductVersionRequest> productVersions);
+    }
+}
```

Comment:
Really small point but should ProductIdentifiers should probably be productIdentifiers (i.e. lower case p)

Reply:
Did respective changes

Reply:
Thanks